### ISSUE TITLE
LLAP protocol spec

### ISSUE THREAD

**Issue by ondrej1024** [CisecoPlc/LLAPSerial#6]
I know that unfortunately WirelessThings had to close down. I have still some XRF modules and sensors lying around and would like to continue using them. But the wirelessthings, ciseco and openmicros web sites are all offline and with them the documentation.
So I was wondering if someone has a copy of the LLAP protocol spec and command reference which he/she could share? This would be really helpful.
Thanks a lot.

**Comment by mrlocke** [comment]
Yep, It's a pity. I also have some modules I'm using. There is a nice thing called internet archive, which thankfully has a snapshot of the openmicros's web: https://web.archive.org/web/20160314065906/http://openmicros.org/index.php/articles/85-llap-lightweight-local-automation-protocol

**Comment by ondrej1024** [comment]
Thanks for the link. It's good to have all the docs. I think LLAP is a great protocol and might use it for some of my projects. I wonder why it hasn't been picked up by others.
